README.md

```diff
@@ -65,3 +73,36 @@ Download and install [nodejs](https://nodejs.org),
 7. Before you create a pull request run the linter. Warnings are ok, but errors should be fixed.
 
 `yarn lint`
+
+### Using docker
```

Comment:
Nice! Could you specify that this is optional, and is only required if you want to run the app in a docker container?

Reply:
Thank you for the notice. I've updated it to be hopefully more clear. Let me know

Reply:
Perfect! Do you have permissions to merge the PR, or should I do it?